packages/babel-plugin/src/utils/ast.tsx

```diff
@@ -69,6 +75,12 @@ export const getMemberExpressionMeta = (expression: t.MemberExpression) => {
     MemberExpression(path) {
       if (t.isIdentifier(path.node.object)) {
         bindingIdentifier = path.node.object;
+        originalBindingType = bindingIdentifier.type;
+      } else if (t.isCallExpression(path.node.object)) {
+        if (t.isIdentifier(path.node.object.callee)) {
+          bindingIdentifier = path.node.object.callee;
+          originalBindingType = path.node.object.type;
```

Comment:
when we want to handle args later - what will we do? any major changes needed?

Reply:
🤔 In such case we just have to capture arguments which are being passed when we are checking for `t.isCallExpression` and if those arguments are used in return statement we will evaluate them. I think we can put them somewhere in the state, probably in metadata and use them later when evaluating return statement? Its just matter of saving arguments somewhere 🤔